pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/TopicStats.java

```diff
@@ -284,6 +286,11 @@ static void printTopicStats(SimpleTextOutputStream stream, String cluster, Strin
                 metric(stream, cluster, namespace, topic, n, c.consumerName(), c.consumerId(),
                         "pulsar_consumer_msg_rate_out", consumerStats.msgRateOut,
                         splitTopicAndPartitionIndexLabel);
+
+                metric(stream, cluster, namespace, topic, n, c.consumerName(), c.consumerId(),
```

Comment:
We also need a namespace level metric since some users do not enable the topic level metrics.
And if the topic level metrics are enabled, we should not the namespace level metric, otherwise, we will get the same indicator name but with different labels.